src/components/Cards/NotificationCard.tsx

```diff
@@ -0,0 +1,104 @@
+import { createSignal } from 'solid-js';
+import ErrorIcon from '~/assets/imgs/svgs/error.svg';
+import InfoIcon from '~/assets/imgs/svgs/info.svg';
+import WarningIcon from '~/assets/imgs/svgs/warning.svg';
+import '~/assets/scss/components/notification-card.scss';
+import { unified } from 'unified';
+import remarkParse from 'remark-parse';
+import remarkFrontmatter from 'remark-frontmatter';
+import remarkParseFrontmatter from 'remark-parse-frontmatter';
+import remarkRehype from 'remark-rehype';
+import rehypeStringify from 'rehype-stringify';
+import content from '~/content/notification.md?raw';
+
+interface NotificationContentDefinition {
+  html: string;
+}
+
+interface Frontmatter {
+  type: string;
+  title: string;
+}
+
+function NotificationContent(props: NotificationContentDefinition) {
+  return <span innerHTML={props.html} />;
+}
+
+const NotificationCard = () => {
+  const [showCard, setShowCard] = createSignal(
+    localStorage.getItem('notificationDismissed') !== 'true'
+  );
+  const showNotification = JSON.parse(
+    import.meta.env.VITE_SHOW_NOTIFICATION || 'false'
+  );
+
+  const {
+    data: { frontmatter },
+    value,
+  } = unified()
+    .use(remarkParse)
+    .use(remarkFrontmatter, ['yaml'])
+    .use(remarkParseFrontmatter)
+    .use(remarkRehype)
+    .use(rehypeStringify)
+    .processSync(content);
+
+  const handleDismiss = () => {
+    localStorage.setItem('notificationDismissed', 'true');
+    setShowCard(false);
+  };
+
+  const typedFrontmatter = frontmatter as Frontmatter;
+
+  const notificationType = typedFrontmatter?.type;
+  const notificationTitle = typedFrontmatter?.title;
+
+  const getNotificationIcon = () => {
+    switch (notificationType) {
+      case 'warning':
+        return (
+          <WarningIcon

```

Comment:
Lets give the icons some color to help them pop more. corn100 for warning fire100 for error and sky100 for info, from the openaq design system.